internal/peer/node/start.go

```diff
@@ -828,9 +834,32 @@ func serve(args []string) error {
 		}()
 	}
 
+	signalFunc := func() {
+		// notify all externally built chaincodes of signal
+		close(externalCCSignalChan)
```

Comment:
Why would you bury this here instead of a defer out of the caller? This makes the function a one-shot and if it's called multiple times, it's going to panic.